modules/nodes-nodes-garbage-collection-configuring.adoc

```diff
@@ -76,6 +76,11 @@ metadata:
 
 . Create a custom resource (CR) for your configuration change.
 +
+[IMPORTANT]
+====
+If there is one file system, or if `/var/lib/kubelet` and `/var/lib/containers/` are in the same file system, the settings with the highest values trigger evictions, as those are met first. These evictions are associated with the system triggering the eviction.
```

Comment:
These evictions are associated with the system triggering the eviction. >> I read the BZ, but this part of the note is unclear. The file system triggers the eviction. Is that what you're saying? I wonder if the passive voice here is making this sentence more confusing than necessary and should be modified to simply "The file system triggers the eviction."

Reply:
yes good point.